client/views/edit/edit.js

```diff
@@ -0,0 +1,168 @@
+  },
+  maxQuestionFifth() {
+    return (Template.instance().data.table.max_question === 450) ? 'selected' : '';
+  },
+  // maxresponse
+  maxResponseFirst() {
+    return (Template.instance().data.table.max_response === 100) ? 'selected' : '';
+  },
+  maxResponseSecond() {
+    return (Template.instance().data.table.max_response === 150) ? 'selected' : '';
+  },
+  maxResponseThird() {
+    return (Template.instance().data.table.max_response === 200) ? 'selected' : '';
+  },
+  maxResponseFourth() {
+    return (Template.instance().data.table.max_response === 250) ? 'selected' : '';
+  },
+  maxResponseFifth() {
+    return (Template.instance().data.table.max_response === 300) ? 'selected' : '';
+  },
+  // new_length
+  newLengthFirst() {
+    return (Template.instance().data.table.new_length === 30) ? 'selected' : '';
+  },
+  newLengthSecond() {
+    return (Template.instance().data.table.new_length === 60) ? 'selected' : '';
+  },
+  newLengthThird() {
+    return (Template.instance().data.table.new_length === 300) ? 'selected' : '';
+  },
+  newLengthFourth() {
+    return (Template.instance().data.table.new_length === 3600) ? 'selected' : '';
+  },
+  newLengthFifth() {
+    return (Template.instance().data.table.new_length === 86400) ? 'selected' : '';
+  },
+  newLengthSixth() {
+    return (Template.instance().data.table.new_length === 604800) ? 'selected' : '';
+  },
+  // stale_length
+  staleFirst() {
+    return (Template.instance().data.table.stale_length === 900) ? 'selected' : '';
+  },
+  staleSecond() {
+    return (Template.instance().data.table.stale_length === 1800) ? 'selected' : '';
+  },
+  staleThird() {
+    return (Template.instance().data.table.stale_length === 3600) ? 'selected' : '';
+  },
+  staleFourth() {
+    return (Template.instance().data.table.stale_length === 86400) ? 'selected' : '';
+  },
+  staleFifth() {
+    return (Template.instance().data.table.stale_length === 604800) ? 'selected' : '';
+  },
+  staleSixth() {
+    return (Template.instance().data.table.stale_length === 2592000) ? 'selected' : '';
+  },
+  staleLengthSeventh() {
+    return (Template.instance().data.table.stale_length === 31557600) ? 'selected' : '';
+  },
+  // hidden
+  visibilityFirst() {
+    return (Template.instance().data.table.hidden === false) ? 'selected' : '';
+  },
+  visibilitySecond() {
+    return (Template.instance().data.table.hidden === true) ? 'selected' : '';
+  },
+  // social
+  socialFirst() {
+    return (Template.instance().data.table.social === true) ? 'selected' : '';
+  },
+  socialSecond() {
+    return (Template.instance().data.table.social === false) ? 'selected' : '';
+  },
+});
+
+Template.edit.events({
+  'click .editsubmitbutton': function (event, template) {
+    const table = Template.instance().data.table;
+    const thresholdSelect = document.getElementsByName('threshold')[0];
+    const threshold = thresholdSelect[thresholdSelect.selectedIndex].value;
```

Comment:
This and the line above it can be reduced to a single line using `Template.$`, which gives you a jQuery object, and then you can use the [jQuery API](https://api.jquery.com/val/) to get the value.